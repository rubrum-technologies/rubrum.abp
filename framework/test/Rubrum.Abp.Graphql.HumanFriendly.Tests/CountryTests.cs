using CookieCrumble;
using Shouldly;
using Xunit;

namespace Rubrum.Abp.Graphql.HumanFriendly;

public class CountryTests : RubrumAbpGraphqlHumanFriendlyTestBase
{
    [Fact]
    public async Task FetchById()
    {
        await using var result = await ExecuteRequestAsync(b => b.SetQuery(
            """
            query {
                countryByHumanFriendlyId(id: 1) {
                    id
                    name
                    humanFriendlyId
                }
            }
            """));

        result.ShouldNotBeNull();

        result.MatchSnapshot();
    }
}
