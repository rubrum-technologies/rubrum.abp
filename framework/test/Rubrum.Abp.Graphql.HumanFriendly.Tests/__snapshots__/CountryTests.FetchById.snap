{
  "data": {
    "countryByHumanFriendlyId": {
      "id": "Q291bnRyeQpnNWE1MzM0MzJlZGRhNDZhZmFkYWEyZTZlNTFlNDYxNjM=",
      "name": "Russian",
      "humanFriendlyId": 1
    }
  }
}